src/Phinx/Config/Config.php

```diff
@@ -49,6 +49,11 @@ class Config implements ConfigInterface
     protected $configFilePath;
 
     /**
+     * @var $phinx_version
+     */
+    public static $phinx_version = "0.4.1";
```

Comment:
No, I don't think so. Dependency Inversion implies that the more dynamic or configurable something is, the **shallower** it should be. See #413 for additional detail on this topic.